### PR TITLE
Update/fortigate forwardedfor

### PR DESCRIPTION
**Pull request by rombernier**
context: https://github.com/SekoiaLab/integration/issues/154

### CODE REVIEW

File: Fortinet/fortigate/ingest/parser.yml [hunk 1]
```diff
@@ -241,6 +241,7 @@ stages:
           fortinet.fortigate.attack.id: "{{parsed_event.message.attackid}}"
           fortinet.fortigate.policyid: "{{parsed_event.message.policyid}}"
           fortinet.fortigate.poluuid: "{{parsed_event.message.poluuid}}"
+          fortinet.fortigate.forwardedforip: "{{parsed_event.message.forwardedfor}}"
```
Comment (squioc) [inline Fortinet/fortigate/ingest/parser.yml:244]: [network.forwarded_ip](https://www.elastic.co/guide/en/ecs/8.11/ecs-network.html#field-network-forwarded-ip) is more suitable:

```suggestion
          network.forwarded_ip: "{{parsed_event.message.forwardedfor}}"
```

File: Fortinet/fortigate/_meta/fields.yml [hunk 2]
```diff
@@ -53,6 +53,11 @@ fortinet.fortigate.event.type:
   name: fortinet.fortigate.event.type
   type: keyword
 
+fortinet.fortigate.forwardedforip:
+  description: Ip for which the event was forwarded
+  name: fortinet.fortigate.forwardedforip
+  type: keyword
+
```
Comment (squioc) [inline Fortinet/fortigate/_meta/fields.yml:60]: According to the suggestion above:
```suggestion
```

**Review by squioc — CHANGES_REQUESTED** [review]
Thank you for the contribution.
For forwarded ip, there is an ECS field: `network.forwarded_ip`.

**Comment by rombernier** [comment]
> Thank you for the contribution. For forwarded ip, there is an ECS field: `network.forwarded_ip`.

Nice, perfect

**Review by squioc — APPROVED** [review]
LGTM

**Pull request merged**